### ISSUE TITLE
p-editor only shows unstyled buttons when it's inside of p-sidebar

### ISSUE THREAD

**Issue by rubenteck** [primefaces/primeng#15297]
### Describe the bug

When placing a p-editor inside a p-sidebar, it doesn't properly render the p-editor. Only the header's buttons are showed without styling.
stackblitz: https://stackblitz.com/edit/github-evzyaq?file=src%2Fapp%2Fapp.component.ts

### Environment

I use a windows 11 laptop with IntelliJ. Angular 17.3.4, PrimeNG 17.13.0 and node v18.20.1.

### Reproducer

https://stackblitz.com/edit/github-evzyaq?file=src%2Fapp%2Fapp.component.ts

### Angular version

17.3.4

### PrimeNG version

17.13.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

v18.20.1

### Steps to reproduce the behavior

1. place a p-editor inside a p-sidebar
2. open the sidebar

### Expected behavior

the p-editor should render correctly and be useable

**Comment by RinoGaetano94** [comment]
Same here passing it to a custom component. Something like:

**my-custom-container.content.xhtml**
```
<div>
  <ng-content />
</div>
```

**foobar.component.xhtml**
```
<my-custom-container>
  <p-editor />
</my-custom-container>
```
Angular 17.3.4
PrimeNG 17.13.0
Node 20.9.0

**Comment by dl3g0** [comment]
Same here passing

**Comment by aaron-cardenas** [comment]
Its the same in dialogs, it seams related to fix of SSR in 17.13.0.

**Comment by rubenteck** [comment]
I can confirm that this was not an issue in PrimeNG 17.12.0 and earlier.
[Stackblitz with PrimeNG 17.12.0](https://stackblitz.com/edit/github-evzyaq-wbswyf) 
(please note that I have also imported quill styles here in the angular.json file)

**Comment by adrianvivas** [comment]
I saw this was fixed in 17.15.0, however, I am using version 17.18.10 and the buttons are unstyled yet
